Networking/Networking/Model/Order.swift

```diff
@@ -194,10 +194,7 @@ public struct Order: Decodable, GeneratedCopiable, GeneratedFakeable {
         // Properties added on WC 6.6, we provide a local fallback for older stores.
         let isEditable = try container.decodeIfPresent(Bool.self, forKey: .isEditable) ?? Self.inferIsEditable(status: status)
         let needsPayment = try container.decodeIfPresent(Bool.self, forKey: .needsPayment) ?? Self.inferNeedsPayment(status: status, total: total)
-
-        // TODO: Update with local fallback when required.
-        // https://github.com/woocommerce/woocommerce/blob/3611d4643791bad87a0d3e6e73e031bb80447417/plugins/woocommerce/includes/class-wc-order.php#L1537-L1561
-        let needsProcessing = try container.decodeIfPresent(Bool.self, forKey: .needsProcessing) ?? false
+        let needsProcessing = try container.decodeIfPresent(Bool.self, forKey: .needsProcessing) ?? Self.inferNeedsProcessing(status: status)
```

Comment:
Do we need an extra function for the default value here? Since the logic is very straightforward, we could use `status == .processing` directly here as default value

Reply:
From what we chatted in Slack, I'll leave `inferNeedsProcessing(status: status)` for the time being, but updated the inline comment to reflect why we're using this approach at the moment. Changed on 7501f8efce535c07dc24ade101ceaa17b977b634